app/src/hooks/usePpg.ts

```diff
@@ -19,6 +20,7 @@ type Hooks = {
     email: string
   ) => Promise<void>;
   verifyPublicKey: (email: string | undefined) => Promise<void>;
+  resendVerificationEmail: (email: string) => Promise<void>;
```

Comment:
```suggestion
  resendVerificationEmail: (email: string) => Promise<boolean>;
```

Reply:
no return from function just a alert box, do you mean API endpoint return boolean?